examples/decoding/plot_decoding_csp_timefreq.py

```diff
@@ -0,0 +1,101 @@
+# Authors: Laura Gwilliams <laura.gwilliams@nyu.edu>
+#          Jean-Remi King <jeanremi.king@gmail.com>
+#          Alex Barachant <alexandre.barachant@gmail.com>
+#          Alexandre Gramfort <alexandre.gramfort@telecom-paristech.fr>
+#
+# License: BSD (3-clause)
```

Comment:
This example needs a docstring (see the related [CircleCI failure](https://circleci.com/gh/mne-tools/mne-python/4154?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link)), check out any of the other examples for the format